lib/split/alternative.rb

```diff
@@ -43,20 +50,21 @@ def participant_count=(count)
 
     def completed_count(goal = nil)
       unless @completed_count
-        @completed_count = HashWithIndifferentAccess.new
-        self.goals.each do |goal|
+        @completed_count = ActiveSupport::HashWithIndifferentAccess.new
+        # add nil so we include overall completed count
+        (self.goals + [nil]).each do |goal|
```

Comment:
Having nil in the array will enable fetching the completed counts for experiments that have no goals. Since the function increment_completion accepts nil goal thus I think this function which returns the counts that increment_completion increments should also accept nil goal.